src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -269,9 +259,24 @@ impl Sandbox for VirtSandbox {
         Ok(())
     }
 
-    async fn cleanup(&self, _id: &str) -> Result<()> {
-        self.resource_manager.delete_cgroups().await?;
-        self.hypervisor.cleanup().await?;
+    async fn cleanup(&self) -> Result<()> {
+        info!(sl!(), "delete cgroup");
+        self.resource_manager
+            .delete_cgroups()
+            .await
+            .context("delete cgroups")?;
+
+        info!(sl!(), "delete hypervisor");
+        self.hypervisor
+            .cleanup()
+            .await
+            .context("delete hypervisor")?;
+
+        info!(sl!(), "clean up host path");
```

Comment:
Is it possible to provide a  `cleanup()` for the resource manager, including the `delete_cgroups` and `cleanup_hostpath`.

Reply:
fixed